airflow/dag_processing/manager.py

```diff
@@ -1254,3 +1078,39 @@ def reload_configuration_for_dag_processing():
     importlib.reload(airflow.settings)
     airflow.settings.initialize()
     del os.environ["CONFIG_PROCESSOR_MANAGER_LOGGER"]
+
+
+def process_parse_results(
+    run_duration: float,
+    finish_time: datetime,
+    run_count: int,
+    path: str,
+    parsing_result: DagFileParsingResult | None,
+    processor_subdir: str | None,
+    session: Session,
+) -> DagFileStat:
```

Comment:
Worth adding a docstring describing it on high-level i.e parsing and store dag and import errors in DB